kornia/losses/tversky.py

```diff
@@ -10,6 +10,49 @@
 # https://github.com/kevinzakka/pytorch-goodies/blob/master/losses.py
 
 
+def tversky_loss(input: torch.Tensor, target: torch.Tensor,
```

Comment:
Why new losses are here? Is it because of some rebase problems? I thought, it should be only GPU tests

Reply:
not new, just refactored some stuff